coalib/results/Result.py

```diff
@@ -63,6 +65,13 @@ def __init__(self,
         if severity not in RESULT_SEVERITY.reverse:
             raise ValueError("severity is not a valid RESULT_SEVERITY")
 
+        self.diffs = {}
+        if(diffs and isinstance(diffs, dict)):
```

Comment:
Redundant brackets:
`if diffs and isinstance(diffs, dict):`

Also, wouldnt `if isinstance(diffs, dict):` be sufficient ?